.gitmodules

```diff
@@ -223,7 +223,7 @@
 [submodule "third_party/silabs/wifi_sdk"]
 	path = third_party/silabs/wifi_sdk
 	url = https://github.com/SiliconLabs/wiseconnect.git
-	branch = v3.3.2
```

Comment:
This isn't necessary to do in matter_sdk, if we want to do this update it should be done in CSA. We use the wifi_sdk from the matter_extension, so we can still comeplete the task to move to 3.3.3 by updating matter_extensions wifi_sdk submodule.